src/development/tools/sdk/release-notes/release-notes-3.0.0.md

```diff
@@ -9,6 +9,67 @@ For information about subsequent bug-fix releases, see
 
 [Hotfixes to the Stable Channel]: https://github.com/flutter/flutter/wiki/Hotfixes-to-the-Stable-Channel
 
+## If you see warnings about bindings
+
+You may see warnings like this:
+```
+Warning: Operand of null-aware operation '!' has type 'SchedulerBinding' which excludes null.
+```
+...when migrating to Flutter 3.
```

Comment:
@sfshaza2 I'm curious what your preferred way of handling a block of code in the middle of a sentence.